gen/dcos-config.yaml

```diff
@@ -1425,7 +1425,7 @@ package:
     content: |
       {
         "PATH": "/usr/bin:/bin",
-        "SHELL": "/usr/bin/bash",
+        "SHELL": "/bin/bash",
```

Comment:
Why are you confident `bash` will be available in `/bin` here, when in `gen/azure/cloud-config.yaml` you have used `/usr/bin/env bash`?

Reply:
I prefer to use `/usr/bin/env bash` in cross-distribution scripts. However, you can't use that for a shell.

```
# CentOS defaults to /usr/bin/bash by PATH
cgianelloni@chris:~/Repos/mine/dcos$ docker run --rm -ti centos:7 bash -c 'type -P bash'
/usr/bin/bash
# However, it's also available as /bin/bash
cgianelloni@chris:~/Repos/mine/dcos$ docker run --rm -ti centos:7 bash -c 'ls -l /bin/bash'
-rwxr-xr-x 1 root root 964608 Oct 30  2018 /bin/bash
# And it's /bin/bash on Ubuntu
cgianelloni@chris:~/Repos/mine/dcos$ docker run --rm -ti ubuntu:18.04 bash -c 'type -P bash'
/bin/bash
```